src/components/Books/Books.js

```diff
@@ -95,10 +102,17 @@ const Books = ({
       .filter((book) => favoriteBooks.includes(book.id))
       .map((book) => <BookItem key={book.id} book={book} />);
 
-  const handleScroll = () => {};
+  const handleScroll = () => {
+    if (
+      window.innerHeight + document.documentElement.scrollTop ===
+      document.scrollingElement.scrollHeight
```

Comment:
It's better to use `document.documentElement.clientHeight` instead of 'window.innerHeight', because 'window.innerHeight' includes `height` of horizontal scroll.